.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,7 @@
 Thanks for submitting a pull request!
 
 Please make sure you've read and understood our contributing guidelines;
-https://github.com/netlify/netlify-cms/blob/master/CONTRIBUTING.md
+https://github.com/netlify/netlify-cli/blob/main/CONTRIBUTING.md
```

Comment:
😱

Reply:
Yep, I guess since it's a comment it went unnoticed